features/messages/impl/src/main/res/values-hu/translations.xml

```diff
@@ -39,16 +39,16 @@
   <string name="screen_room_timeline_reactions_show_more">"Több megjelenítése"</string>
   <string name="screen_room_timeline_read_marker_title">"Új"</string>
   <plurals name="screen_room_timeline_state_changes">
-    <item quantity="one">"%1$d szobaváltozás"</item>
-    <item quantity="other">"%1$d szobaváltozás"</item>
+    <item quantity="%1$d szobaváltozás">"one"</item>
+    <item quantity="%1$d szobaváltozás">"other"</item>
   </plurals>
   <plurals name="screen_room_typing_many_members">
-    <item quantity="one">"%1$s, %2$s és %3$d további felhasználó"</item>
-    <item quantity="other">"%1$s, %2$s és %3$d további felhasználó"</item>
+    <item quantity="%1$s, %2$s és %3$d további felhasználó">"one"</item>
+    <item quantity="%1$s, %2$s és %3$d további felhasználó">"other"</item>
   </plurals>
   <plurals name="screen_room_typing_notification">
-    <item quantity="one">"%1$s éppen ír…"</item>
-    <item quantity="other">"%1$s éppen ír…"</item>
+    <item quantity="%1$s éppen ír…">"one"</item>
+    <item quantity="%1$s éppen ír…">"other"</item>
```

Comment:
Hello @meskobalazs do you know what happened here? The new format does not work.

Reply:
It seems that it was a temporary error, syncing again the strings fixes it...

Reply:
Actually @pixlwave has reverted all the changes...

Reply:
I have no idea how that happened, it could be a bug on Localazy's end. The only atypical thing I might do is using POEdit's cloud editor feature.